package.json

```diff
@@ -5,37 +5,40 @@
   "module": "src/index.ts",
```

Comment:
How does this work, does `main` and `module` get added to the package.json that actually gets published?

Reply:
ugh, that's no good

Reply:
wait, no, this is ok - it's there for codesandbox.  Pay no attention to the /package.json, just to lib/package.json